src/theme/DocSidebar/index.js

```diff
@@ -1,12 +1,15 @@
 import React, {useState, useCallback} from 'react';
-import classnames from 'classnames';
+
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useLockBodyScroll from '@theme/hooks/useLockBodyScroll';
 import useLogo from '@theme/hooks/useLogo';
 import Link from '@docusaurus/Link';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import isInternalUrl from '@docusaurus/isInternalUrl';
 
+import WithBackgroundImage from 'components/WithBackgroundImage';
+
+import classnames from 'classnames';
```

Comment:
For this and future PRs, you might see me reshuffling the imports... This is because I'm changing things to be in line with Docusaurus's import order:

External Modules
Docusuarus imports
My React Components
CSS related

Docusuarus doesn't follow this in 100% of their files, but like 80%

Reply:
Great, I like that ordering!